tests/report/host/host.cpp

```diff
@@ -28,7 +28,7 @@ extern int FileToBytes(const char* path, std::vector<uint8_t>* output);
 
 void generate_and_save_report(oe_enclave_t* enclave)
 {
-#ifdef OE_USE_LIBSGX
+#ifdef OE_HAS_QUOTE_PROVIDER
```

Comment:
It looks like this test we changed to remove testing local attestation without DCAP. Why? That's breaking the feature beyond what I would have expected necessary for this changeset (i.e. only the removal of `oe_get_report` for remote quote on AESM should be the only dev facing change).

Reply:
Added back local attestation tests. It looks like we have to initialize target_info to 0s before passing it into `GetReport_v2`.